templates/zerver/api/server-settings.md

```diff
@@ -23,49 +23,7 @@
 
 #### Return values
 
-* `authentication_methods`: object in which each key-value pair in the object
-  indicates whether the authentication method is enabled on this server.
-* `zulip_version`: the version of Zulip running in the server.
-* `zulip_feature_level`: an integer indicating what features are
-    available on the server. The feature level increases monotonically;
-    a value of N means the server supports all API features introduced
-    before feature level N.  This is designed to provide a simple way
-    for client apps to decide whether the server supports a given
-    feature or API change.  See the [changelog](/api/changelog) for
-    details on what each feature level means.
-
-    **Changes**.  New in Zulip 2.2.  We recommend using an implied value
-    of 0 for Zulip servers that do not send this field.
```

Comment:
This `zulip_feature_level` text is better and should be ported to `zulip.yaml`

Reply:
Done